### ISSUE TITLE
More advanced translation scopes

### ISSUE THREAD

**Issue by pavelbonda** [excid3/noticed#130]
Hi!

Currently when I use translation method `t('.message')` in class `Admin::SomeNotification` it returns following translation:
```yml
notifications: 
  admin/some_notification:
    message: Some
```
This approach does not benefit from `yaml` nesting and leads to some duplication. It would be good to have an option to use following translation path for described case:
```yml
notifications:
  admin:
    some_notification:
      message: Some
```

**Comment by excid3** [comment]
I am not familiar enough with Rails I18n to know why it wouldn't generate the right key there.

Feel free to send a PR!

**Comment by excid3** [comment]
Looks like we'd just need to change the `underscore` here: https://github.com/excid3/noticed/blob/master/lib/noticed/translation.rb#L17

Maybe it needs to be `underscore.gsub("/", ".")` ?

**Comment by pavelbonda** [comment]
Looks correct to me, but it will be a breaking change for some applications, so it requires a major version change 😬

**Comment by excid3** [comment]
I guess we could do it as a minor bump with a note in the changelog, but you're right that it technically should be a major.  😬

**Comment by excid3** [comment]
Fixed by #131
